addons/spreadsheet_dashboard/views/spreadsheet_dashboard_views.xml

```diff
@@ -45,4 +45,19 @@
         </field>
     </record>
 
+    <record id="spreadsheet_dashboard_view_form" model="ir.ui.view">
+        <field name="name">spreadsheet.dashboard.view.form</field>
+        <field name="model">spreadsheet.dashboard</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <field string="Name" name="name"/>
+                    <field string="Dashboard Group" name="dashboard_group_id"/>
+                    <field string="Data" name="data"/>
+                    <field string="Thumbnail" name="thumbnail"/>
+                    <field string="Group" name="group_ids"/>
```

Comment:
`string` attribute is not needed here. It's managed automatically by the framework based on the field
```suggestion
                    <field name="name"/>
                    <field name="dashboard_group_id"/>
                    <field name="data"/>
                    <field name="thumbnail"/>
                    <field name="group_ids"/>
```